### DISCUSSION TITLE
How Can disable Kafka connect sanitizer

### DISCUSSION THREAD

**Discussion by jcmonsalve2021** [provectus/kafka-ui#1701] [Q&A] [answered]
Hi All , I need disable the KC sanitizer , I tried with env   KAFKA_CONFIG_SANITIZER_ENABLED =false , but doesnt work ... What is the correct way to do this ??

**Comment by Haarolean** [comment, marked as answer]
You're probably running an outdated version. This config property has been introduced on 1st of February. You can try either `master`-labeled image or any fixed one from docker hub.
